tests/paramgrill.c

```diff
@@ -1087,6 +1468,17 @@ static U8** createMemoTableArray(ZSTD_compressionParameters paramConstraints, co
     return mtAll;
 }
 
+static ZSTD_compressionParameters maskParams(ZSTD_compressionParameters base, ZSTD_compressionParameters mask) {
```

Comment:
minor naming detail : 

a `mask` is generally something different : 
it's typically a bitmap, which selectively enable / disable each target field, using as low as one bit per signal.

Here, this is more about a vector `ZSTD_compressionParameters` which may contain optional parameters.
In this case, they overwrite existing parameters.

Another name like `new` or equivalent would better translate the intention.